### ISSUE TITLE
Having trouble running examples/possible geopy issue

### ISSUE THREAD

**Issue by anna-02** [fhamborg/Giveme5W1H#61]
I am getting the following error message when running parse_single_from_code.py:

Traceback (most recent call last):
  File "/Users/annaablove/Desktop/py_test/who.py", line 53, in <module>
    extractor = MasterExtractor()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/Giveme5W1H/extractor/extractor.py", line 60, in __init__
    environment_extractor.EnvironmentExtractor(),
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/Giveme5W1H/extractor/extractors/environment_extractor.py", line 55, in __init__
    self.geocoder = Nominatim(domain=host, timeout=8)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/geopy/geocoders/nominatim.py", line 103, in __init__
    Nominatim(user_agent="my-application")
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/geopy/geocoders/nominatim.py", line 103, in __init__
    Nominatim(user_agent="my-application")
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/geopy/geocoders/nominatim.py", line 103, in __init__
    Nominatim(user_agent="my-application")
  [Previous line repeated 487 more times]
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/geopy/geocoders/nominatim.py", line 92, in __init__
    super().__init__(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/geopy/geocoders/base.py", line 242, in __init__
    self.adapter = adapter_factory(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/geopy/adapters.py", line 335, in __init__
    self.session = requests.Session()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/sessions.py", line 367, in __init__
    self.headers = default_headers()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/utils.py", line 821, in default_headers
    return CaseInsensitiveDict({
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/structures.py", line 46, in __init__
    self.update(data, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/_collections_abc.py", line 854, in update
    if isinstance(other, Mapping):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/abc.py", line 98, in __instancecheck__
    return _abc_instancecheck(cls, instance)
RecursionError: maximum recursion depth exceeded in comparison

I think it has something to do with Nominatim.py, but I am inexperienced with geopy and am having trouble isolating the problem. Any help would be appreciated, thanks!

**Versions (please complete the following information):**
 - OS: MacOS 10.15.6
 - Python Version 3.9

**Comment by ajyl** [comment]
TL;DR
you can probably do 
```
pip3 uninstall geopy
pip3 install geopy==1.13.0
```

For repo authors, I think `setup.py` needs to be updated - I'm seeing that `setup.py` will install `geopy>=1.11.0`, which gives me `2.1.0`. However, it seems like geopy versions > 2.0 introduces breaking changes. I saw that in `requirements.txt` that `geopy==1.13.0` is specified. I think `setup.py` needs to be updated accordingly.
